src/app/components/activity/feed/devlog-post/text/text.ts

```diff
@@ -13,22 +17,22 @@ import { ActivityFeedView } from '../../view';
 	components: {
 		AppFadeCollapse,
 		AppContentViewer,
+		AppLoading,
 	},
 })
 export default class AppActivityFeedDevlogPostText extends Vue {
-	@Inject()
-	feed!: ActivityFeedView;
+	@Inject() feed!: ActivityFeedView;
 
-	@Prop(ActivityFeedItem)
-	item!: ActivityFeedItem;
+	@Prop(ActivityFeedItem) item!: ActivityFeedItem;
+	@Prop(FiresidePost) post!: FiresidePost;
 
-	@Prop(FiresidePost)
-	post!: FiresidePost;
-
-	isToggling = false;
+	isLoaded = !!this.post.article_content;
```

Comment:
rename to more specific `isArticleLoaded`. Also, we can set it to true even if article_content isn't populated if we check against has_article, yeah?

Reply:
This component is literally just a component to show the article. I would normally agree, but since it's scoped to just working with the article, I don't think we need to name it longer.

Also, has_article doesn't mean that article content is loaded in. It means that the article in the DB has content. I should probably change this check to use `post.hasArticleContent` which works off the loaded content to check.